### ISSUE TITLE
Declare support for Python 3.9

### ISSUE THREAD

**Issue by hugovk** [hugovk/pypistats#171]
Python 3.9.0 is due out today.

* https://www.python.org/dev/peps/pep-0596/

Let's declare support by adding the 3.9 Trove classifier to `setup.py`.

---

When doing this, if 3.9.0 is available on Travis CI or GitHub Actions (i.e. `3.9`, not `3.9-dev`), also update/add `3.9` to `.travis.yml` or `.github/workflows/test.yml`, 

But please test it actually works by enabling the CI on your fork first.

**Comment by liisberg** [comment]
I would like to help out with this, please assign this issue to me.

**Comment by liisberg** [comment]
3.9 is not yet found in the [manifest file](https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json). Please see the actions in the [forked repository](https://github.com/liisberg/pypistats/actions/runs/289354614).

I was able to get it working locally on Ubuntu 20.04 by adding ppa:deadsnakes/ppa and installing python3.9.
Reproducing it locally will require the following steps:
```
$ curl -O https://bootstrap.pypa.io/get-pip.py
$ python3.9 get-pip.py
$ pip3 install python-dateutil==2.8.1
```
And then finally we can run pypistats:

```
$ pypistats recent pillow
| last_day | last_month | last_week |
|---------:|-----------:|----------:|
|  398,287 | 19,021,728 | 3,950,409 |
```

Will you accept this pull request for now or wait untill the manifest file is updated?

Thanks.

**Comment by hugovk** [comment]
Thanks for checking, let's just go for the `setup.py` changes for now.

---

I expect it'll be available on the CIs some days after the actual 3.9.0 release.
